sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/integrationtest/BeamSqlStringFunctionsIntegrationTest.java

```diff
@@ -31,8 +31,8 @@ public void testStringFunctions() throws Exception {
             .addExpr("CHARACTER_LENGTH('hello')", 5)
             .addExpr("UPPER('hello')", "HELLO")
             .addExpr("LOWER('HELLO')", "hello")
-            .addExpr("POSITION('world' IN 'helloworld')", 5)
-            .addExpr("POSITION('world' IN 'helloworldworld' FROM 7)", 10)
+            .addExpr("POSITION('world' IN 'helloworld')", 6)
+            .addExpr("POSITION('world' IN 'helloworldworld' FROM 7)", 11)
```

Comment:
our array indices are 0-based, is it ok to make strings 1-based?

Reply:
Pretty much every SQL is 1 based.